xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESound.h

```diff
@@ -59,6 +61,7 @@ class CActiveAESound : public IAESound
   static int64_t Seek(void *h, int64_t pos, int whence);
 
 protected:
+  CActiveAE *m_activeAE;
```

Comment:
noted but this is not you decision here. AE is a rather self contained component and this code has no influence on anly other code outside AE. for this round I'll keep it as is